app/services/application_key_backend_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module ApplicationKeyBackendService
+  module_function
+  extend ApplicationAssociationBackendService
+
+  def reflection_name
+    :application_keys
+  end
+
+  def pisoni_class
+    ThreeScale::Core::ApplicationKey
```

Comment:
also in here, better to define the class in a string way and then constantine the string to a class.

it could end up failing on a pre-loading class error etc. this should be a standard so let's ask @guicassolato and @hallelujah too.

Reply:
> it could end up failing on a pre-loading class error etc

In this case, isn't it a good thing? because we would notice right away as soon as we make any change in Pisoni (like upgrading it) that it is not working 🤔

Reply:
Although maybe you are right and it is better to make it fail only at that moment 🤔 I dunno 🤔

Reply:
ok, leave it as it is :) we can discuss it later.